vkcs/vpnaas/resource_site_connection.go

```diff
@@ -161,12 +165,51 @@ func ResourceSiteConnection() *schema.Resource {
 				ValidateDiagFunc: networking.ValidateSDN(),
 				Description:      "SDN to use for this resource. Must be one of following: \"neutron\", \"sprut\". Default value is project's default SDN.",
 			},
+			"traffic_selector_ep_merge": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"sdn"},
+				Description:  "This argument controls whether multiple traffic selection rules for a network interface should be merged into one rule, or applied independently. When set to \"true\", this parameter can improve system performance, while providing more granular control over traffic selection when set to \"false\". __note__ Available only in sprut SDN.",
```

Comment:
lets add that default value is true

Reply:
I can’t do this, it will break the creation of the Neutron network

Reply:
in the note

Reply:
done